.idea/bc-ta-application-system.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+  <component name="TestRunnerService">
+    <option name="projectConfiguration" value="Nosetests" />
+    <option name="PROJECT_TEST_RUNNER" value="Nosetests" />
+  </component>
+</module>
```

Comment:
I don't know what they do but they're almost definitely from PyCharm because these files show in explorer but not in PyCharm. I think they have to do with some built-in git functionality PyCharm offers.

Reply:
Oh ok, gotcha. In order to avoid pushing these files to github, can you add ".idea" to the .gitignore file? Then also be sure to remove these files from the branch so we don't merge them into master.